File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <sstream>
#include <algorithm>

using namespace std;

string solution(string s) {
    stringstream ss(s); // 문자열 공백으로 자르기
    vector<int> nums;
    int num;
    
    while(ss >> num) {     // string -> int 바로 받기
        nums.push_back(num);
    }
    
    sort(nums.begin(), nums.end());
    
    return to_string(nums.front()) + " " + to_string(nums.back());   // 정수 -> 문자열
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        int[] nums = Arrays.stream(s.split(" ")).mapToInt(Integer::parseInt).toArray();
    
        Arrays.sort(nums);
        
        StringBuilder sb = new StringBuilder();
        sb.append(nums[0]).append(" ").append(nums[nums.length - 1]);
        
        return sb.toString();
    }
}